auto_tests/forwarding_test.c

```diff
@@ -0,0 +1,318 @@
+/*
+ * Copyright © 2019 The TokTok team.
```

Comment:
nit: use the new SPDX format of copyright comments.

Reply:
I just deleted the copyright block, because the other auto_tests seem not to have one.